Demo/CMakeLists.txt

```diff
@@ -1,21 +1,16 @@
 cmake_minimum_required(VERSION 3.20)
 
-project(RenderingEngineDemo LANGUAGES C CXX)
 
-set(Dependencies ${CMAKE_CURRENT_LIST_DIR}/../dependencies)
-set(BuildSystem ${Dependencies}/sequoia/build_system)
-set(DemoDir ${CMAKE_CURRENT_LIST_DIR}/../Demo)
-set(TestingUtils ${CMAKE_CURRENT_LIST_DIR}/../TestingUtilities)
+add_executable(Demo)
 
-include(${BuildSystem}/Utilities.cmake)
-
-sequoia_init()
-
-add_executable(Demo DemoMain.cpp)
+target_sources(Demo
+	PRIVATE
+		DemoMain.cpp
+)
 
 target_include_directories(Demo PRIVATE ${DemoDir})
 
-add_subdirectory(${TestingUtils}/curlew curlew)
+# TODO: Demo depends on test utilities - Inversion of dependency!
```

Comment:
The Demo project is just another form of testing, so I think the dependency is the right way around

Reply:
Makes sense in a manner and this is only an experiment, I had expected the Demo to be essentially an 'example' app that can be built without test functionality. At present I see this is not possible as the core manager type is defined in TestUtilities.